tests/core/snapshots/default-value-input-type.md_0.snap

```diff
@@ -0,0 +1,15 @@
+---
+source: tests/core/spec.rs
+expression: response
+---
+{
+  "status": 200,
+  "headers": {
+    "content-type": "application/json"
+  },
+  "body": {
+    "data": {
+      "abc": 1
```

Comment:
did you create this snapshot manually? because it's kinda right

Reply:
I actually generated this snapshot by manually passing `{id: 1}` and then changed the query and marked the test skipped